server/db/dbManager.ts

```diff
@@ -323,3 +323,30 @@ export class UserManager {
 		this.model.remove({}).exec();
 	}
 }
+
+export class JobManager {
+	model: mongoose.Model<models.IJob>;
+
+	constructor() {
+		this.model = mongoose.model('Jobs', models.JobSchema);
+	}
+
+	async addJob(data: Partial<models.IPost>, fn: Function): Promise<mongoose.Types.ObjectId> {
+		return (await this.model.create({
+			data,
+			value: fn.toString()
+		}))._id;
+	}
+
+	deleteJob(id: mongoose.Types.ObjectId): void {
+		this.model.deleteOne({ _id: id }).exec();
+	}
+
+	async getAllJobs() {
+		return await this.model.find().exec();
+	}
+
+	async DELETE_ALL_JOBS() {
+		this.model.remove({}).exec();
+	}
```

Comment:
Couple of comments here (they may be implemented to other methods in this class):
1. Do you really need `async` here? Think about where it should be used and why.
2. Let me assume that you used caps naming because of the importance of the method. Commonly, caps naming is used for constant. So if you don't have any other reason to make the difference in naming, I wouldn't suggest doing this at all. It is better to add a comment.
3. The class is named `JobManager` and you use `Job` for each method... Is it worth truncating the naming to make it shorter and actionable?

```suggestion
	deleteAll() {
		this.model.remove({}).exec();
	}
```